conans/client/remover.py

```diff
@@ -37,7 +37,14 @@ def remove(self, conan_ref):
         self.remove_packages(conan_ref)
         self._remove(self._paths.export(conan_ref), conan_ref, "export folder")
         self._remove(self._paths.export_sources(conan_ref), conan_ref, "export_source folder")
-        self._remove(self._paths.conan(conan_ref), conan_ref)
+        conan_ref_dir = self._paths.conan(conan_ref)
+        self._remove(conan_ref_dir, conan_ref)
+        try:
+            dir_name, channel = os.path.split(conan_ref_dir)
+            os.remove(os.path.join(dir_name, "%s.count" % channel))
```

Comment:
Locks file name details shouldn't be propagated.

Reply:
Yes, I know, but I haven't found a better way. Any suggestion?

Reply:
Use a method in the paths to knowing the name of the locks.